docker-compose.prod.yml

```diff
@@ -1,37 +1,44 @@
 version: '3.7'
 
 services:
-  web:
+  web_prod:
     build:
       context: ./app
-      dockerfile: Dockerfile.prod
-    command: gunicorn app.wsgi:application --bind 0.0.0.0:8000
+      dockerfile: Dockerfile
+    command: ["--production"]
     volumes:
-      - static_volume:/home/app/web/app/static
-      - media_volume:/home/app/web/app/media
+      - static_volume_prod:/home/app/web/app/static
+      - media_volume_prod:/home/app/web/app/media
     expose:
       - 8000
     env_file:
-      - ./.env.prod
+      - ./.env
+    environment:
+      DEBUG: 0
+      DJANGO_SETTINGS_MODULE: app.settings.production
     depends_on:
       - db
-  db:
+
+  db_prod:
```

Comment:
Should this just be db? Or is this so you could run a db and db_prod image at the same time? Currently it breaks the default environment variable for SQL_HOST.

Reply:
Yes, the idea was to have separate dev and prod containers, so I added an override of the `SQL_HOST` value in 1d67ccf.